src/rt-app_args.c

```diff
@@ -67,11 +71,13 @@ parse_command_line(int argc, char **argv, rtapp_options_t *opts)
 	int c;
 
 	while (1) {
-		c = getopt_long(argc, argv, "hv", long_args, 0);
+		c = getopt_long(argc, argv, "hvl:", long_args, 0);
 		if (c == -1)
 			break;
 
 		switch (c) {
+		case -1:
```

Comment:
Isn't this caught by the `if (c == -1)` above?

Reply:
yes, it is...